modules/monitoring-enabling-query-logging-for-thanos-querier.adoc

```diff
@@ -6,8 +6,7 @@
 [id="enabling-query-logging-for-thanos-querier_{context}"]
 = Enabling query logging for Thanos Querier
 
-[role="_abstract"]
-For default platform monitoring in the `openshift-monitoring` project, you can enable the Cluster Monitoring Operator to log all queries run by Thanos Querier.
+For default platform monitoring in the `openshift-monitoring` project, you can enable the {cmo-first} to log all queries run by Thanos Querier.
```

Comment:
```suggestion
For default platform monitoring in the `openshift-monitoring` project, you can enable the {cmo-full} to log all queries run by Thanos Querier.
```

Reply:
@juzhao Thank you for the comment. I made some small changes as I was applying the attribute as some of the original text was incorrect and did not follow our guidelines. For example, here, because it is first mention of the Operator, it should have been referred to as "Cluster monitoring Operator (CMO)" so I corrected some of these issues by using the right form for that situation (using {CMO-first} instead of {CMO-full} in this case). I hope that explains the decitions :+1: